client/hosting-overview/components/plan-card.tsx

```diff
@@ -142,7 +187,7 @@ const PlanCard: FC = () => {
 								plain
 								href={ `/add-ons/${ site?.slug }` }
 							>
-								{ translate( 'Need more storage?' ) }
+								{ translate( 'Need more space?' ) }
```

Comment:
I think we should stick with "storage" since that's how we advertise the feature in wordpress.com/plans and wordpress.com/pricing

Reply:
I'm convinced, will change back to storage cc: @lucasmendes-design